hack/make/.integration-test-helpers

```diff
@@ -25,7 +25,7 @@ run_test_integration() {
 	done
 
 	(
-		flags="-check.v -check.timeout=${TIMEOUT} -test.timeout=360m $TESTFLAGS"
+		flags="-check.v -check.timeout=${TIMEOUT} -test.timeout=360m -check.f TestLogsFollowSlow $TESTFLAGS"
```

Comment:
Looks like this needs to be removed.

You can set this in `TESTFLAGS` instead of modidying this file:

`TESTFLAGS="-check.f TestLogsFollowSlow"`